lib/AST/ASTContext.cpp

```diff
@@ -1835,6 +1835,8 @@ ASTContext::getSpecializedConformance(Type type,
   auto result
     = new (*this, arena) SpecializedProtocolConformance(type, generic,
                                                         substitutions);
+  auto node = specializedConformances.FindNodeOrInsertPos(id, insertPos);
+  assert(!node);
```

Comment:
You'll get an unused variable warning on this in a no-asserts build.

Reply:
Oh, right thanks!